tests/storage/lvm_test.py

```diff
@@ -1038,17 +1038,18 @@ def stale_pv(tmp_storage):
     pvs = sorted(pv.name for pv in lvm.getAllPVs())
```

Comment:
Note for the commit message:

Using --config is fine, and it does not overwrite system configuration,
it overwrites the configuration for the particular lvm command. The problem
is that we use lvm devicesfile by default, and when a system is configured
to use devicsfile you cannot use --config 'devices { filter = ...}'.

So this updates the tests so they can run on a host configured for vdsm, using
lvm devicesfile by default.

I'm not sure how this work in the container before and after the change, but we can dig deeper later.